src/main/dynamo/ScanBuilder.ts

```diff
@@ -0,0 +1,89 @@
+import { JayZ } from "@ginger.io/jay-z"
+import { DynamoDB } from "aws-sdk"
+import { DocumentClient } from "aws-sdk/clients/dynamodb"
+import { QueryExpressionBuilder } from "./expressions/QueryExpressionBuilder"
+import { groupModelsByType } from "./groupModelsByType"
+import {
+  groupAllPages,
+  IteratorOptions,
+  pagedIterator,
+  PaginatedQueryResults,
+} from "./pagedIterator"
+import { Table } from "./Table"
+import { GroupedModels, TaggedModel } from "./types"
+
+type ScanConfig<T extends TaggedModel> = {
+  db: DynamoDB.DocumentClient
+  table: Table
+  jayz?: JayZ
+  consistentRead?: boolean
+}
+
+/** Builds and executes parameters for a DynamoDB Scan operation */
```

Comment:
And here's our class to build scans -- it largely mirrors our `QueryBuilder` class.